official/recommendation/uplift/layers/uplift_networks/base_uplift_networks.py

```diff
@@ -33,7 +34,7 @@ class BaseTwoTowerUpliftNetwork(tf_keras.layers.Layer, metaclass=abc.ABCMeta):
   def call(
       self,
       inputs: types.DictOfTensors,
-      training: bool | None = None,
-      mask: tf.Tensor | None = None,
+      training: Union[bool, None] = None,
```

Comment:
I think `Union[something | None]` is equivalent to `Optional[something]`. Could you update here and all the other places if applicable.